SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -8,10 +8,13 @@
 #include "LoadBalancing/AbstractLBStrategy.h"
 #include "LoadBalancing/OwnershipLockingPolicy.h"
 #include "Schema/AuthorityIntent.h"
+#include "Schema/MigrationDiagnostic.h"
 #include "Schema/SpatialDebugging.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialLoadBalancingHandler);
 
+using namespace SpatialGDK;
```

Comment:
This doesn't look needed

Reply:
Doesn't build without it because MigrationDiagnostic is in namespace SpatialGDK

Reply:
Okay - please remove other direct usages of SpatialGDK in this file then

Reply:
Yes sure made it consistent.